src/app/clusters/resource-monitoring-server/resource-monitoring-server.cpp

```diff
@@ -232,6 +242,32 @@ CHIP_ERROR Instance::EnumerateAcceptedCommands(const ConcreteClusterPath & clust
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR Instance::ReadReplacableProductList(AttributeValueEncoder & aEncoder)
+{
+    CHIP_ERROR err;
+    if (Instance::HasFeature(ResourceMonitoring::Feature::kReplacementProductList))
+    {
+        ReplacementProductListManager * productListManagerInstance = Instance::GetReplacementProductListManagerInstance();
```

Comment:
Why does this need the Instance:: prefix?  This is not a static method.... and we are inside Instance anyway.